test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryStoreTest.cs

```diff
@@ -23,5 +23,31 @@ protected override void AddRoleStore(IServiceCollection services, object context
         {
             services.AddSingleton<IRoleStore<IdentityRole>, InMemoryRoleStore<IdentityRole>>();
         }
+
+        protected override InMemoryUser CreateTestUser(string username = "", string email = "",
```

Comment:
Yeah we could do that. I'll InMemory user altogether. There is a TestUser also I believe. Can I remove that too ?